admin/src/Events.elm

```diff
@@ -176,6 +192,7 @@ createEvent event toMsg =
     apiRequest
         { method = Post
         , url = [ "events" ]
+        , queries = queryAll
```

Comment:
or define a `apiRequestWithoutQuery` with an empty `queries` field. This would require an additional function definition but avoids the repetitive `queryAll` use.

Reply:
I agree that `queryAll` is a bad name, but specifying the queries is necessary.

Reply:
Will fix in #49.